language/move-command-line-common/src/address.rs

```diff
@@ -66,6 +66,11 @@ impl NumericalAddress {
                 bytes: address,
                 format: NumberFormat::Ss58,
             });
+        } else if let Ok(address) = move_vm_support::base58_address::base58_to_move_address(s) {
```

Comment:
since we have an unconditional return above, the `if-else` is not needed here
```suggestion
        }

        if let Ok(address) = move_vm_support::base58_address::base58_to_move_address(s) {
```